packages/@aws-cdk/aws-s3/README.md

```diff
@@ -441,6 +441,16 @@ new s3.Bucket(this, 'MyBucket', {
 });
 ```
 
+### Bucket owner enforced (recommended)
```

Comment:
At the introduction of object ownership, a few paragraphs above, it reads:

> You can use the two following properties (...)

Can you change it to something like

> You can use one of the following properties (...)

Reply:
Good catch. Thanks!
Updated.